### DISCUSSION TITLE
Issue on Huawei with Lam Alif diacritical marks

### DISCUSSION THREAD

**Discussion by riazpatel** [harfbuzz/harfbuzz#4647] [Q&A] [answered]
Hey everyone. I'm having a strange issue that seems to only occur on Huawei devices - the arabic letter lam alif (لا) does not render it's diacritical marks (harakaat) correctly on Huawei phones, no issue on Samsung phones though.

I've attached an image showing the issue. In Android studio it renders correctly however on the Huawei devices it seems to jump from the Lam portion of the letter to the Alif part.

Any assistance would be greatly appreciated!

Riaz

![Capture](https://github.com/harfbuzz/harfbuzz/assets/5467664/7e1a3e15-295d-4227-ae0e-e496638e4dc6)

**Comment by behdad** [comment, marked as answer]
Those are two different fonts. The Huawei font doesn't have ligature attachments. So it's basically a font shortcoming.

**Reply by riazpatel** [reply]
Thank you Behdad! Your assistance is very much appreciated. I am now using the Noto Naskh Arabic as you suggested and it works perfectly :)
